docs/source/tutorial.rst

```diff
@@ -46,6 +46,22 @@ body of response reveals the information about the created auction: its internal
 modified. Pay attention to the `procurementMethodType`. Note that auction is
 created with `active.tendering` status.
 
+Keep in mind that `startDate` of `auctionPeriod` can't be closer than seven days 
+to current date. In other words lot exposition minimal period must be at least 
```

Comment:
Please, check the second part (....can't be closer than seven days to current date).
Lexically, it has a rather vague sense.